python_modules/dagster/dagster/seven/yaml.py

```diff
@@ -0,0 +1,17 @@
+# pylint: disable=unused-import
+from __future__ import absolute_import
+
+from functools import partial
+
+from yaml import dump as dump_
+from yaml import load as load_
+from yaml import FullLoader
```

Comment:
This will fail on older versions of PyYAML. I suspect we should just use `yaml.safe_load` instead of `yaml.load(Loader=...)` -- this is a semantic change, but we never intended to support YAML serialization of anything other than literals (YAML representations are intended to be interchangeable with JSON/GraphQL query throughout).

Reply:
@mgasner thank you for the suggestion, can you please re-review?